### PR TITLE
Phaser poc

### PR DESCRIPTION
**Pull request by dolanmiu**

### CODE REVIEW

File: src/main/client/scene/main-menu-scene.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   }
+
+   create(): void {
+      this.add
+         .text(100, 50, "This is a sample main menu. Click the \"Start\" button below to run your game.", {
+            fill: "#FFFFFF"
+         })
+         .setFontSize(24);
+
+      new MenuButton(this, 100, 150, "Start Game", () => {
```
Comment (dolanmiu) [inline src/main/client/scene/main-menu-scene.ts]: not sure if i like this, 

it creates a new button, then doesn't use the instantiated object

Comment (barnabasbartha) [reply src/main/client/scene/main-menu-scene.ts]: Sure, I just copied it from an example repo, feel free to change the ui.

File: src/main/client/game/game-component.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class GameComponent {
       });
    }
 
-   refreshScale() {
+   refreshScale(): void {
```
Comment (barnabasbartha) [inline src/main/client/game/game-component.ts:35]: I suggest write every return type except void.

**Review by barnabasbartha — CHANGES_REQUESTED** [review]

File: src/main/client/game/game-component.ts (hunk 2 above)
Comment (barnabasbartha) [reply src/main/client/game/game-component.ts:35]: Discussed, we'll use void too.

**Review by barnabasbartha — APPROVED** [review]

File: src/main/client/scene/main-menu-scene.ts (hunk 1 above)
Comment (dolanmiu) [reply src/main/client/scene/main-menu-scene.ts]: nice

**Pull request merged**